### PR TITLE
chore(cli-installer): readme improvements

### PR DESCRIPTION
**Pull request by DanielHougaard**
# Description 📣

This PR contains some simple README improvements for our new NPM CLI Installer, to reduce confusion between the CLI and Node SDK

## Type ✨

- [ ] Bug fix
- [ ] New feature
- [x] Improvement
- [ ] Breaking change
- [ ] Documentation

---

- [x] I have read the [contributing guide](https://infisical.com/docs/contributing/getting-started/overview), agreed and acknowledged the [code of conduct](https://infisical.com/docs/contributing/getting-started/code-of-conduct). 📝

### CODE REVIEW

File: npm/README.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
-<h1 align="center">Infisical</h1>
+<h1 align="center">Infisical CLI</h1>
 <p align="center">
-  <p align="center"><b>The open-source secret management platform</b>: Sync secrets/configs across your team/infrastructure and prevent secret leaks.</p>
+  <p align="center"><b>The Infisical CLI is powerful command line tool that can be used to retrieve, modify, export and inject secrets into any process or application as environment variables.</b></p>
```
Comment (scott-ray-wilson) [inline npm/README.md:3]: typo: missing a -> `is a powerful command...`

but I would suggest making this top section more grabby and then using the bottom section to be more explicit/technical; maybe something like this?

"Embrace shift-left security with Infisical CLI and strengthen your DevSecOps practices by seamlessly managing secrets across your workflows, pipelines and applications."

File: npm/README.md [hunk 2]
```diff
@@ -36,10 +36,7 @@
 
 ### Introduction
 
-**[Infisical](https://infisical.com)** is the open source secret management platform that teams use to centralize their application configuration and secrets like API keys and database credentials as well as manage their internal PKI.
-
-We're on a mission to make security tooling more accessible to everyone, not just security teams, and that means redesigning the entire developer experience from ground up.
-
+The Infisical CLI is powerful command line tool that can be used to retrieve, modify, export and inject secrets into any process or application as environment variables. You can use it across various environments, whether it’s local development, CI/CD, staging, or production.
```
Comment (scott-ray-wilson) [inline npm/README.md:39]: same typo; missing a -> `is a powerful command...`

thoughts on something like:

"The Infisical CLI streamlines secret management by letting you retrieve, modify, and inject environment variables across your entire development pipeline. From local development to production deployments, it integrates seamlessly with your applications and workflow."

**Review by scott-ray-wilson — CHANGES_REQUESTED** [review]
Two minor typos but also some suggestions!

File: npm/README.md (hunk 1 above)
Comment (DanielHougaard) [reply npm/README.md:3]: Implemented!

File: npm/README.md (hunk 2 above)
Comment (DanielHougaard) [reply npm/README.md:39]: Let's stick with what we have right now (with the typo fixed). I've copied this directly from our CLI documentation actually. I will update the typo in our docs as well

**Review by scott-ray-wilson — DISMISSED** [review]

**Review by scott-ray-wilson — APPROVED** [review]
LGTM!

**Pull request merged**